package.json

```diff
@@ -97,7 +100,6 @@
     "eslint-plugin-unicorn": "^48.0.1",
     "eslint-plugin-vitest": "^0.3.8",
     "fast-glob": "^3.3.2",
-    "glob": "^10.3.10",
     "globby": "^14.0.1",
```

Comment:
why do we need 2 glob libs?

Reply:
Explained why I went with `glob` over `fast-glob` in https://github.com/code-pushup/cli/pull/839#discussion_r1800915711

The other glob libs were here as dev dependencies because of performance benchmarks (not sure it makes sense to have them in main branch for that reason only, though). None of them were being used in any of our packages until now, so I moved `glob` to dependencies now that there's a production use case.